### ISSUE TITLE
Preloader is very imperceptible

### ISSUE THREAD

**Issue by EvgeniyDoctor** [jellyfin/jellyfin-web#4594]
**Describe The Bug**
Preloader is very imperceptible on the IPTV page. For example, i click on the channel and i can't always see if it's loading.
And i am pretty sure this is not the only place where the preloader has this problem.

**Steps To Reproduce**

1. set IPTV list
2. play it

**Expected Behavior**

Preloader should be clearly visible and noticeable. It should has another color, for example.

**Logs**

**Screenshots**

Try to find the preloader here:
![Снимок экрана (11)](https://github.com/jellyfin/jellyfin-web/assets/15073200/6adf32ce-5ac5-4724-b4c2-28e0cb4d7089)

**System (please complete the following information):**
 - Platform: Linux
 - Browser: Brave
 - Jellyfin Version: 10.8.10

**Additional Context**

**Comment by jsuelwald** [comment]
Yeah, same here and i agree, this should be very different

**Comment by plafreniere** [comment]
You could just add custom CSS, for example : 

```
.docspinner {
  background : rgba(0,0,0,0.6);
  box-shadow: 0px 0px 50px rgba(0,0,0,0.6);
  border-radius: 100px;
}
```

**Comment by EvgeniyDoctor** [comment]
@plafreniere yeah, and thousands of users, of course, will be happy to find this instruction and to do the same. And do it again, if some user installs Jellyfin on another computer.

**Comment by Sukhendu2002** [comment]
Can you assign this issue to me I want to work on this

**Comment by thornbill** [comment]
We don't typically assign issues to people. Anyone is free to work on it.

**Comment by Sukhendu2002** [comment]
Got it
